src/docs/product/performance/trace-view.mdx

```diff
@@ -36,11 +36,13 @@ The transaction view provides a split view similar to the [span view](/product/p
 
 ### Orphan Traces and Broken Subtraces
 In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
+As well broken subtraces can occur when sentry received an error for a trace, but the corresponding transaction does not exist. This means that the error will not appear on the trace view, and the quick trace will be empty.
```

Comment:
I'm not sure if "the quick trace will be empty" is clear. Do you mean the Quick Trace element won't show up at all or some part of it will be empty? If it's the first, that needs to be explicit and if it's the latter, a screenshot might be needed to illustrate that.

Reply:
It won't show up at all would this be better?
```suggestion
As well broken subtraces can occur when sentry received an error for a trace, but the corresponding transaction does not exist. This means that the error will not appear on the trace view, and the quick trace will not show up.
```

Reply:
Suggest the following for clarity:

...and the quick trace won't appear on the **Event Details** or **Issue Details** pages.

Reply:
:+1: its only for issue details, but updated in 61b92d6